lib/ansible/cli/scripts/ansible_cli_stub.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Was this supposed to change?

Reply:
it wsan't... Snuck in there from my testing.